pkg/reconciler/v1alpha1/configuration/queueing_test.go

```diff
@@ -136,11 +137,8 @@ func newTestController(t *testing.T, servingObjects ...runtime.Object) (
 
 func TestNewConfigurationCallsSyncHandler(t *testing.T) {
 	config := getTestConfiguration()
-	// TODO(grantr): inserting the configuration at client creation is necessary
-	// because ObjectTracker doesn't fire watches in the 1.9 client. When we
-	// upgrade to 1.10 we can remove the config argument here and instead use the
-	// Create() method.
-	_, _, servingClient, controller, kubeInformer, servingInformer := newTestController(t, config)
+	stopCh := make(chan struct{})
```

Comment:
is this ever closed?

Reply:
Yes in the defer below. I'll move it closer.

Reply:
Ah, I thought this was new